### PR TITLE
add release 1.9.3

### PR DESCRIPTION
**Pull request by andreaturli**

### CODE REVIEW

**Comment by jclouds-commentator** [comment]
Go to http://59ae4980266a1e5a8dd0-9e8e45a35acd365ec99ece5f99f23a2b.r31.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://acc9c1cd1493c1d03e6d-781c7d5238af67dd6e2d4e0cc3db7e2b.r83.cf5.rackcdn.com/ to review your changes.

File: releasenotes/1.9.3.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+You can read the official announcement at [Apache jclouds 1.9.3 released](http://markmail.org/message/lyozqjxkcxm47zh5). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
+
+This is a bugfix release. The following list shows the fixed issues and improvements:
+
+#### Bug
```
Comment (nacx) [inline releasenotes/1.9.3.md]: Most of these have not the corresponding JIRA issues. How about rewording this to:

*This is a bugfix release. The following list shows the fixed issues and patches that are included in the release:*
*#### Bugs and patches*

Comment (andreaturli) [inline releasenotes/1.9.3.md]: thx I'll update that

**Comment by jclouds-commentator** [comment]
Go to http://876692b0ccf23192b3a9-f6bbbd5b8b5404b10b4dae7843b61d46.r2.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://8da327ec2faf8b0dd3f7-abfe5deef6647c31080f7054d3e2afb1.r93.cf5.rackcdn.com/ to review your changes.

File: releasenotes/1.9.3.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+You can read the official announcement at [Apache jclouds 1.9.3 released](http://markmail.org/thread/qlapnppmfbilje7p). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
+
+This is a bugfix release. The following list shows the fixed issues and patches that are included in the release:
+
+#### Bug and patches
```
Comment (demobox) [inline releasenotes/1.9.3.md]: "Bugs"?

File: releasenotes/1.9.3.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+* <a href='https://github.com/jclouds/jclouds/pull/1027'>PR-1027</a> - fix jclouds/apis/swift retryOnRenew
+* <a href='https://github.com/jclouds/jclouds/pull/991'>PR-991</a> - Fix for ssh reload for CentOS 7
+* <a href='https://issues.apache.org/jira/browse/JCLOUDS-1044'>JCLOUDS-1044</a> - fix handling NULL JsonTokens in adapters under NullFilteringTypeAdapterFactories class
+* <a href='https://github.com/jclouds/jclouds/pull/956'>PR-956</a> - update CloudstackTemplateOptions pattern
+* <a href='https://github.com/jclouds/jclouds/pull/937'>PR-937</a> - fix hpcloud compute adapter
+* <a href='https://github.com/jclouds/jclouds/pull/934'>PR-934</a> - Version range specification for guava in OSGi
+* <a href='https://github.com/jclouds/jclouds/pull/932'>PR-932</a> - fix for issue if SL machine has multiple credentials registered
+* <a href='https://github.com/jclouds/jclouds/pull/907'>PR-907</a> - Fix: RunScriptOptions equals
```
Comment (demobox) [inline releasenotes/1.9.3.md]: Just curious why we have so many fixes without corresponding JIRA issues here?

File: releasenotes/1.9.3.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+## <a id="credits"></a>Credits
+
+jclouds would like to thank everyone who contributed time and effort in order to make this release happen:
```
Comment (demobox) [inline releasenotes/1.9.3.md]: "The Apache jclouds team"?

File: releasenotes/1.9.3.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+* Apache jclouds [PMC](http://people.apache.org/committers-by-project.html#jclouds-pmc) and community for verifying the release.
```
Comment (demobox) [inline releasenotes/1.9.3.md]: "The Apache ..."?

File: releasenotes/index.md [hunk 6]
```diff
@@ -7,6 +7,7 @@ permalink: /releasenotes/
 The Release Notes and Javadocs for every major release of Apache jclouds are listed chronologically below.
 
 * [2.0.0](/releasenotes/2.0.0) | [Javadoc](/reference/javadoc/2.0.x/)
+* [1.9.2](/releasenotes/1.9.3) | [Javadoc](/reference/javadoc/1.9.x/)
```
Comment (demobox) [inline releasenotes/index.md]: "1.9.3"? ;-)

**Comment by jclouds-commentator** [comment]
Go to http://47ea4e8a996157d05ad1-c393c86521b0d7767f466e64c39494f3.r51.cf5.rackcdn.com/ to review your changes.

File: releasenotes/1.9.3.md (hunk 3 above)
Comment (andreaturli) [inline releasenotes/1.9.3.md]: we should better in tracking bugs on jira

**Comment by andreaturli** [comment]
@demobox can you have another shot at it?

File: releasenotes/index.md (hunk 6 above)
Comment (nacx) [inline releasenotes/index.md]: @andreaturli This is still wrong

**Comment by andreaturli** [comment]
thanks @nacx, good spot!

**Comment by jclouds-commentator** [comment]
Go to http://1d547f9a57b7cf99a2cc-3d89fc21b654ea514b9e52407d648ba8.r20.cf5.rackcdn.com/ to review your changes.

**Comment by demobox** [comment]
Changes look good - thanks @andreaturli! Just two related points:

* Javadoc still needs to be published? http://jclouds.apache.org/reference/javadoc/1.9.x/ points at 1.9.2
* http://jclouds-javadocs.elasticbeanstalk.com/ still shows 2.0.0-SNAPSHOT docs - is that broken? I guess it should be 2.0.1-SNAPSHOT now

**Comment by nacx** [comment]
>http://jclouds-javadocs.elasticbeanstalk.com/ still shows 2.0.0-SNAPSHOT docs - is that broken? I guess it should be 2.0.1-SNAPSHOT now?

I'll have a look at this

**Comment by demobox** [comment]
> http://jclouds-javadocs.elasticbeanstalk.com/ still shows 2.0.0-SNAPSHOT docs - is that broken? I guess it should be 2.0.1-SNAPSHOT now?

It looks like that one is actually [on me](https://github.com/demobox/jclouds-aggregate-javadocs/commit/a11b49340c68ea093b021c785fe4e0a6c9031678) - working on it...

/cc @nacx

**Comment by nacx** [comment]
Uops :) I've just edited the build and changed the labs providers to those that make sense. All yours from the current point.

**Comment by demobox** [comment]
> I've just edited the build and changed the labs providers to those that make sense

Just looked at that config diff ;-) All looks good to me; current build is still running. Thanks, @nacx!

**Comment by demobox** [comment]
> All looks good to me; current build is still running

Ah, I see you just killed it ;-) Will kick off another one...

**Comment by andreaturli** [comment]
thanks @demobox and @nacx, merging now!
